src/intl/messages.js

```diff
@@ -566,10 +584,15 @@ export const messages: { [messageId: string]: MessageType } = {
   selectTheBootableDeviceTooltip: 'Select the bootable devices in the order you would like them to boot.',
   sendShortcutKey: 'Send Key',
   sendCtrlAltDel: 'Ctrl+Alt+Del',
+  serialConsole: 'Serial Console',
+  serialConsoleOptions: 'Serial Console Options',
+  serverStorage: 'Server-side',
+  serverStorageTooltip: 'Settings are persisted on the server.',
```

Comment:
This should be considered in the context of locale settings only

Reply:
Just wanted to cover all cases here. We have actually 3 options:
1. server-side - persisted as part of user profile i.e. refresh interval
2. client side - persisted in local storage i.e. locale
3. in-memory - not persisted at all - i.e. notifications

Reply:
Let's have a separate discussion regarding this. It think that it might confuse users and not really needed to have those options. It might also complex our solution. 
For most settings - option 1 should be enabled unless we didn't implement this yet (notifications) or unless we have gaps that might break existing functionality like the locale.
option 2 is an internal solution since we also persisted the locale on server as well.